src/components/ppom/transactions.js

```diff
@@ -0,0 +1,307 @@
+import globalContext from '../..';
+import { maliciousAddress } from '../../sample-addresses';
+import {
+  ERC20_SAMPLE_CONTRACTS,
+  ERC721_SAMPLE_CONTRACTS,
+  MALICIOUS_CONTRACT_ADDRESSES,
+} from '../../onchain-sample-contracts';
+import { isBaseNetworkId, isSepoliaNetworkId } from '../../utils';
+
+export function ppomMaliciousTransactionsAndSignatures(parentContainer) {
+  parentContainer.insertAdjacentHTML(
+    'beforeend',
+    `<div
+      class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12 d-flex align-items-stretch"
+    >
+      <div class="card full-width">
+        <div class="card-body">
+          <h4 class="card-title">
+            PPOM - Malicious Transactions and Signatures
+          </h4>
+          <p>We know we are vulnerable if any of these Transactions/Signatures are not flagged as Malicious</p>
+          <h5>Transactions</h5>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousRawEthButton"
+            disabled
+          >
+            Malicious Eth Transfer
+          </button>
+          <button
+          class="btn btn-primary btn-lg btn-block mb-3"
+          id="mintSepoliaERC20"
+          hidden
+        >
+          Mint ERC20
+        </button>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousERC20TransferButton"
+            disabled
+            title="This will only be flagged if you have some ERC20 balance"
+          >
+            Malicious ERC20 Transfer (USDC)
+          </button>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousApprovalButton"
+            disabled
+          >
+            Malicious ERC20 Approval (BUSD)
+          </button>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousContractInteractionButton"
+            disabled
+          >
+              Malicious Contract Interaction
+          </button>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousSetApprovalForAll"
+            disabled
+          >
+            Malicious Set Approval For All
+          </button>
+          <h5>Signatures</h5>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousPermit"
+            disabled
+          >
+            Malicious Permit
+          </button>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousTradeOrder"
+            disabled
+          >
+            Malicious Trade Order
+          </button>
+          <button
+            class="btn btn-primary btn-lg btn-block mb-3"
+            id="maliciousSeaport"
+            disabled
+          >
+            Malicious Seaport
+          </button>
```

Comment:
nit: this section is placed under ppom/transactions but those are actually signatures

Reply:
Yes, this is how the test-dapp was originally organized. We could rename transactions.js to something else.

Reply:
so in this case, when we built this, we created 2 subsections one for Transactions and one for Signatures. They were in the same card, but If that's problematic with the current organization we could have one card for TX and a separate one for Signatures. What do you think?

![Captura desde 2025-01-22 08-51-29](https://github.com/user-attachments/assets/e4c97c2f-b114-4316-8dae-e266e2a7879b)